projects/lib-shared-modules/src/lib/components/dialog-model/dialog-model.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="flex flex-row justify-between items-center bg-primary h-10">
+  <mat-label mat-dialog-title class="text-white">{{dialogueData?.header}}</mat-label>
+  <img class="cursor-pointer pr-2" [mat-dialog-close]="false" src="./../assets/icons/cross-icon.png" alt="">
+</div>
+<div mat-dialog-content class="flex flex-row items-center space-x-4 py-0 text-secondary p-4">
+  <mat-form-field class="w-full mt-4">
+    <mat-label class="text-secondary">{{ dialogueData?.labelname | translate}}</mat-label>
+    <input matInput [(ngModel)]="dialogueData.resourceName">
```

Comment:
why do we need input here ? is this resource upload dialog ?
if it's for resource only, doesn't it will show for other for input dialogues as well ?

Reply:
this is dialog-model is only using for the add-learning resource so only this input tag is added.